### PR TITLE
9 translation of messages using external api

### LINKED ISSUE #9: Translation of messages using external api
https://github.com/LibreTranslate/LibreTranslate

### PR DESCRIPTION
**Pull request by altmannk**
### Related Issue(s)
Closes #9 

### Proposed Changes
- Loads two languages for 'libretranslate' (en, sv)
- LibreTranslateService-class interacts with LibreTranslate API for language detection and translation. Sends POST requests to '/detect' and '/translate' endpoints. The methods will retry the operation if it fails due to [Retryable](https://www.baeldung.com/spring-retry)-annotation.
- 'translateMessage' in WebController handles GET-request to translate title and message for specific message by its ID and renders to display the translated message 
- Each message in '/web/messages' have a 'translate'-link to get the message translated. When pressed it will lead you to a new page, '/web/messages/translate', where the message is translated. There is a return button to take you back to messages.

### Short Description
- service 'libretranslate' in compose.yaml
- LibreTranslateService-class
- GET method 'translateMessage' in WebController-class
- follows same design for translated message, with a 'go back'-button in translatemessage.html
- 'translation'-link functionality for each message placed in messages.html 
- new requestMatcher, '/web/messages/translate*', for authentication in SecurityConfig-class
- supplement with some words in messages.properties and messages_sv.properties

### CODE REVIEW

**Review by Win-ther — DISMISSED** [review]
I really like the placement of the Translate button/link! 
Great job, this is really good! 😁✨

File: src/main/resources/templates/translatemessage.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Translated GilfMessage</title>
```
Comment (marcusju23) [inline src/main/resources/templates/translatemessage.html:4]: Cursed or blessed title? 😄

**Review by marcusju23 — DISMISSED** [review]
vincent-price-looks-good-to-me

File: src/main/resources/templates/translatemessage.html (hunk 1 above)
Comment (altmannk) [reply src/main/resources/templates/translatemessage.html:4]: Hö hö ^^ Ofc its Gilfoyle from Silicon Valley --> [Clean code challenge](https://youtu.be/qNj_sI6qndw?feature=shared&t=31)

image

Comment (marcusju23) [reply src/main/resources/templates/translatemessage.html:4]: Blessed.

**Review by Frlundsten — APPROVED** [review]
Looks good 😄

**Review by Win-ther — APPROVED** [review]

**Review by marcusju23 — APPROVED** [review]

**Pull request merged**